01-Fundamentals-part-2/assignments2.js

```diff
@@ -21,4 +21,22 @@ console.log("\n");
 console.log("Testing Data 2:");
 avgDolphins = calcAverage(85, 54, 71);
 avgKoalas = calcAverage(23, 34, 27);
-checkWinner(avgDolphins, avgKoalas);
+checkWinner(avgDolphins, avgKoalas);*/
+
+function calcTip(billValue) {
+  if (50 < billValue < 300) {
```

Comment:
this way does not work (you can test it in console by typing 50<400<300 e.g.), you have to write conditions separately combining them with AND `&&` or OR `||`